packages/validup-vue/test/unit/composables.spec.ts

```
@@ -273,4 +273,97 @@ describe('useFieldValidation', () => {
+            distinctFields: boolean,
+            distinctForms: boolean,
+        } | undefined;
+
+        const wrapper = mount(defineComponent({
+            setup() {
+                const a = useValidup(container, formState);
+                const b = useValidup(container, reactive({ email: '', name: '' }));
+
+                probe = {
+                    // Same field identity → same bundle.
+                    sameField: useFieldValidation(a.fields.email) === useFieldValidation(a.fields.email),
```

Biome (2.4.16)

[error] This comparison uses the same expression on both sides.

(lint/suspicious/noSelfCompare)